docs/community-resources.md

```diff
@@ -0,0 +1,26 @@
+---
+id: community-resources
+title: Community Resources
+sidebar_label: Community Resources
+---
+
+Members of the PayID community have created useful resources for using and learning about PayID.
+
+- [PayID Youtube Tutorial by Ken Carabello](https://www.youtube.com/watch?v=gpLJp1lylns)
+
+- The [PayID Validator](https://payidvalidator.com/) is an [open-source project](https://github.com/rswarthout/payid-validator) that validates responses using [JSON schemas](https://docs.payid.org/payid-interfaces) and verifies whether your PayID server follows various [best practices](payid-best-practices), like:
+
+  - [HTTP response headers](payid-headers)
+  - [Cache-Control](payid-best-practices#cache-control)
+  - [CORS accessibility](payid-best-practices#set-cors-cross-origin-resource-sharing-headers)
+  - Response time
+
+  See also: [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator).
+
+- [PayID Server](https://github.com/RockHoward/django-payid-server) - A Django-based PayId reference server.
+
+- [Assorted PayID Client Libraries in Node.js](https://www.npmjs.com/search?q=payid)
```

Comment:
I wouldn't link the NPM search query directly.

Reply:
Will change the wording. How about: 
Search for PayID Client Libraries in [npm](https://www.npmjs.com)

Reply:
I don't think it's useful to link to NPM at all. I think we should have a curated list of resources that are reasonably high-quality, which just searching NPM won't give us.

Reply:
Removed